test/jdk/java/nio/file/Path/MacPathTest.java

```diff
@@ -34,14 +34,26 @@
  *        jdk.test.lib.process.*
  *        TestUtil MacPath
  * @run main MacPathTest
+ * @run main/othervm -Djdk.nio.path.useNormalizationFormD=true MacPathTest
  */
 
 import jdk.test.lib.process.ProcessTools;
 
 public class MacPathTest {
+    private static final String FILE_NAME_ENCODING_PROPERTY =
+        "jdk.nio.path.useNormalizationFormD";
+    private static final boolean ENABLE_FILE_NAME_ENCODING =
+        Boolean.getBoolean(FILE_NAME_ENCODING_PROPERTY);
 
     public static void main(String args[]) throws Exception {
-        ProcessBuilder pb = ProcessTools.createTestJvm(MacPath.class.getName());
+        ProcessBuilder pb;
+        if (ENABLE_FILE_NAME_ENCODING) {
+            String option = ENABLE_FILE_NAME_ENCODING ?
```

Comment:
Hello Brian, it appears that the additional ternary check here is unecessary since the enclosing `if (ENABLE_FILE_NAME_ENCODING)` already implies `ENABLE_FILE_NAME_ENCODING` is `true`.

Reply:
Indeed that is lame. Thanks for catching it. Fixed in b420d2cea162ef7a3abde0522d687a290fe154f2.